api/proto/external.proto

```diff
@@ -488,7 +488,11 @@ message MintConfigTx {
 /// A mint-config transaction coupled with the data used to validate it.
 message ValidatedMintConfigTx {
     MintConfigTx mint_config_tx = 1;
-    Ed25519SignerSetV1 signer_set = 2;
+
+    oneof signer_set {
+        Ed25519SignerSetV1 signer_set_v1 = 2;
+        Ed25519SignerSetV2 signer_set_v2 = 4;
```

Comment:
This is using tag 4 because it needs to be compatible with a change to [`MintConfig`](https://github.com/mobilecoinfoundation/mobilecoin/blob/61ab72847c38695714a263144ae089a5a41cef2d/api/proto/external.proto#L454).